common/script_values/wc_magic_perks_values.txt

```diff
@@ -0,0 +1,26 @@
+﻿wc_magic_lifestyle_focus_auto_selection_weight_base_value = 200
+wc_magic_lifestyle_focus_auto_selection_weight_add_to_base_value = 1800
+wc_magic_lifestyle_perk_focus_selection_weight_multiply_common_value = {
```

Comment:
Why it says perk but used only in focuses? What is point of common?

Reply:
> Why it says perk but used only in focuses?

Fixed

> What is point of common?

Common multiplier of all focuses, added comment in file

Reply:
> Common multiplier of all focuses, added comment in file

But you don't have not-common values.

Reply:
> But you don't have not-common values.

And? (；′⌒`)

What I'm supposed to do with this info?

Remove it and use the same code in all focuses instead of a single common value that I can change just once to change it for all focuses? No, that's plain wrong, so I guess it's not what you meant.

Then remove `common` from the name? Why would I want to do that? I guess it's not what you meant either.

Add not-common values to focuses? Fine, I'll do it later.

Reply:
> What I'm supposed to do with this info?

Remove `common` from the name. It doesn't need to distinct values.

Reply:
> Remove common from the name. It doesn't need to distinct values.

Why? I need it to understand what this thing do from its name and not confuse it with something else.

Reply:
> Why? I need it to understand what this thing do from its name and not confuse it with something else.

All values from this file are common. When I noticed the common label I wondered where is an exception and why other not-common multipliers aren't in this file to make it clear what is different.

Reply:
Done